include/pm/device.h

```diff
@@ -24,79 +24,52 @@ extern "C" {
 
 struct device;
 
-/** @def PM_DEVICE_STATE_ACTIVE
- *
- * @brief device is in ACTIVE power state
- *
- * @details Normal operation of the device. All device context is retained.
- */
-#define PM_DEVICE_STATE_ACTIVE          1
-
-/** @def PM_DEVICE_STATE_LOW_POWER
- *
- * @brief device is in LOW power state
- *
- * @details Device context is preserved by the HW and need not be
- * restored by the driver.
- */
-#define PM_DEVICE_STATE_LOW_POWER       2
-
-/** @def PM_DEVICE_STATE_SUSPEND
- *
- * @brief device is in SUSPEND power state
- *
- * @details Most device context is lost by the hardware.
- * Device drivers must save and restore or reinitialize any context
- * lost by the hardware
- */
-#define PM_DEVICE_STATE_SUSPEND         3
-
-/** @def PM_DEVICE_STATE_FORCE_SUSPEND
- *
- * @brief device is in force SUSPEND power state
- *
- * @details Driver puts the device in suspended state after
- * completing the ongoing transactions and will not process any
- * queued work or will not take any new requests for processing.
- * Most device context is lost by the hardware. Device drivers must
- * save and restore or reinitialize any context lost by the hardware.
- */
-#define PM_DEVICE_STATE_FORCE_SUSPEND	4
-
-/** @def PM_DEVICE_STATE_OFF
- *
- * @brief device is in OFF power state
- *
- * @details - Power has been fully removed from the device.
- * The device context is lost when this state is entered, so the OS
- * software will reinitialize the device when powering it back on
- */
-#define PM_DEVICE_STATE_OFF             5
-
-/** @def PM_DEVICE_STATE_RESUMING
- *
- * @brief device is resuming to active state.
- *
- * @details - The device was previously suspended and is now
- * transitioning to become ACTIVE.
- */
-#define PM_DEVICE_STATE_RESUMING             6
-
-/** @def PM_DEVICE_STATE_SUSPENDING
- *
- * @brief device is suspending.
- *
- * @details - The device is currently transitioning from ACTIVE
- * to SUSPEND.
- */
-#define PM_DEVICE_STATE_SUSPENDING             7
+/** @brief Device power states. */
+enum pm_device_state {
```

Comment:
why are you removing all the detailed comments

Reply:
They have just been shortened and essential information kept.